hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalogs.java

```diff
@@ -34,6 +36,7 @@ private HiveCatalogs() {
   public static HiveCatalog loadCatalog(Configuration conf) {
     // metastore URI can be null in local mode
     String metastoreUri = conf.get(HiveConf.ConfVars.METASTOREURIS.varname, "");
-    return CATALOG_CACHE.get(metastoreUri, uri -> new HiveCatalog(conf));
+    return CATALOG_CACHE.get(metastoreUri, uri -> (HiveCatalog) CatalogUtil.loadCatalog(HiveCatalog.class.getName(),
+        "hive", ImmutableMap.of(), conf));
```

Comment:
Minor: I find this line wrapping very awkward because at a glance and from indentation, it looks like these args are passed to `CATALOG_CACHE.get`. But they are actually inside the lambda and passed to `loadCatalog`.

When breaking lines, I prefer to try not to just break wherever a line gets too long and instead break at reasonable points for readability. If possible, I think this should break at `CatalogUtil.loadCatalog` so that the entire method call is on one line. Otherwise, putting all args on one line makes the most sense so that at least the previous line ends with `loadCatalog(` and it is clear that the next line starts the argument list.

Same with the other changes above. Those aren't as bad as this, but I'd still wrap a little differently (not after the first method arg) to increase readability.

Reply:
agreed, fixed.